Source/dead.cpp

```diff
@@ -93,7 +93,7 @@ void __cdecl InitDead()
 				v10 = monster[v9].MType;
 				v11 = (char *)(v8 - 8);
 				v15 = (int *)8;
-				v14 = &v10->Anims[4].Frames[1];
+				v14 = v10->Anims[4].Frames;

```

Comment:
Is this correct? The previous version took the address of the 1th element, while the second version takes the element of the 0th element.

Reply:
It is indeed correct. The actual code still generates the same byte-for-byte, but the offset in the struct shifted. It was incorrectly based on the psx version, which does not have the CMem field.
```
struct AnimStruct // before
{
	int Frames[9]; // 0x0
	int Rate; // 0x24
	int Delay; // 0x28
};
struct AnimStruct // after (fixed)
{
	int CMem; // 0x0
	int Frames[8]; // 0x4
	int Rate; // 0x24
	int Delay; // 0x28
};
```

Reply:
Ah, great